.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
```

Comment:
1. The code now uses the `actions/github-script@v4.0.2` action to create release notes, replacing the previous method.
2. The `check_commits` job has been modified to properly append commit details with a newline and remove trailing newlines from `COMMIT_DETAILS`.
3. The `commit_details` output has been added to the `check_commits` job.
4. The `create-confluence-page` job now uses the `COMMIT_DETAILS` from the `version-tag` output.
5. The `RELEASE_NOTES` variable in the `create-confluence-page` job has been updated to use `COMMIT_DETAILS` rather than the `RELEASE_NOTES` from the output.